assets/dev/js/editor/document/elements/component.js

```diff
@@ -23,6 +23,10 @@ export default class Component extends ComponentBase {
 					parentIsInner = parentModel.get( 'isInner' ),
 					draggedIsInner = childModel.get( 'isInner' );
 
+				if ( elementor.hooks.applyFilters( 'elementor/editor/element/is-valid-child', false, childModel, parentModel ) ) {
```

Comment:
Try to move to the view

Reply:
https://github.com/elementor/elementor/pull/16252/commits/b124ca604c2a4947b4cbcb5210808cb75e266d17